src/main/java/it/pn/frontend/e2e/pages/destinatario/DestinatarioPage.java

```diff
@@ -58,5 +68,34 @@ public boolean isDateBoxInvalid(){
         return invalidBoxDate;
     }
 
+    public void clickButtonNotificheOnSideMenu(String nomeDelegante){
+        logger.info("verifica bottone notifiche nel layout");
+
+        getWebDriverWait(10).until(ExpectedConditions.visibilityOf(this.sideItemNotificheButton));
+        sideItemNotificheButton.click();
+
+        String id = "side-item-" + nomeDelegante;
+        By buttonNotificheOnSideMenu = By.id(id);
+        getWebDriverWait(10).withMessage("bottone notifiche nel layout non visibile").until(ExpectedConditions.visibilityOfElementLocated(buttonNotificheOnSideMenu));
+        this.js().executeScript("arguments[0].click()", this.element(buttonNotificheOnSideMenu));
 
+    }
+
+    public void clickSulDettaglioNotificaDelegante(){
+        WebElement singolaNotificaDelegante = listaNotificheDelegante.get(0);
+        getWebDriverWait(10).withMessage("la singola notifica del delegante non è visibile").until(ExpectedConditions.visibilityOf(singolaNotificaDelegante));
+        logger.info("Si clicca sulla notifica del delegante");
+        singolaNotificaDelegante.click();
+    }
+
+
+    public void checkDettaglioNotificaPage() {
+        try {
+            getWebDriverWait(10).withMessage("i dati del dettaglio notifica non sono visibili").until(ExpectedConditions.visibilityOf(datiDettaglioNotifica));
+            logger.info("dettaglio notifica caricato correttamente");
+        } catch (TimeoutException e){
+            logger.error("Dettaglio notifica non caricato con errore: " + e.getMessage());
+            Assert.fail("Dettaglio notifica non caricato con errore: " + e.getMessage());
+        }
+    }
```

Comment:
Already exist this one:
https://github.com/pagopa/pn-frontend-e2e/blob/a2b5c5ecf097c4d47bc0e822d6b155b502797449/src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java#L34-L61